Source/WarpX.H

```diff
@@ -186,10 +188,15 @@ public:
     std::unique_ptr<amrex::Parser> Ezfield_parser;
 
     // Algorithms
+    //! Integer that corresponds to the current deposition algorithm (Esirkepov, direct, Vay)
     static long current_deposition_algo;
+    //! Integer that corresponds to the charge deposition algorithm (only standard deposition)
     static long charge_deposition_algo;
+    //! Integer that corresponds to the field gathering algorithm (energy-conserving, momentum-conserving)
     static long field_gathering_algo;
+    //! Integer that corresponds to the particle push algorithm (Boris, Vay, Higuera-Cary)
     static long particle_pusher_algo;
```

Comment:
By the way, is there a reason why these integers are `long` and not simply `int`?

Reply:
no need fr that here, even `int` is too large for what we need.